dev/ci/scripts/utils/parse_yaml.py

```diff
@@ -43,8 +41,9 @@ def yq(yamlfile, key):
         The value of the specified key in the yaml file.
     """
 
-    data = AttrDict(HOMEgfs=_top)
-    data.update({'HOMEgfs': _top})
+    HOMEgfs = find_homegfs()
+    data = AttrDict(HOMEgfs=HOMEgfs)
+    data.update({'HOMEgfs': HOMEgfs})
```

Comment:
This line is redundant as line 45 is providing the value.  Updating to the same value is redundant.